src/routes/upload/+page.svelte

```diff
@@ -205,20 +191,12 @@
                                     />
                                 </button>
                             </td>
-                            <td>
-                                <span class="md:text-[1vw]">{file.name}</span>
-                            </td>
-                            <td>
-                                <span class="hidden text-[1vw] md:flex">[DIRECTORY]</span>
-                                <span class="flex text-base md:hidden">[DIR]</span>
-                            </td>
-                            <td class="">
-                                <span class="hidden text-[1vw] md:flex">{file.lastModified}</span>
-                            </td>
-
-                            <td>
-                                <span class="md:text-[1vw]">{prettyBytes(file.size)}</span>
-                            </td>
+                            <!-- name  -->
+                            {#each [name, type, last_modified, size] as table_item}
+                                <td>
+                                    <span class="md:text-[1vw]">{table_item}</span>
```

Comment:
![Screenshot from 2023-06-26 23-16-05](https://github.com/tokitou-san/CoreProject-V3-UI/assets/114811070/c5247374-5a66-4909-80c7-75ced21768a8)

Need to hide `last_modified` in mobile

Reply:
I changed things.